libs/vkd3d/device.c

```diff
@@ -8302,8 +8302,7 @@ static void d3d12_device_caps_init_shader_model(struct d3d12_device *device)
          * - Work graphs
          *   - uhh...
          */
-        if (device->d3d12_caps.max_shader_model == D3D_SHADER_MODEL_6_7 &&
-                (vkd3d_config_flags & VKD3D_CONFIG_FLAG_ENABLE_EXPERIMENTAL_FEATURES))
+        if (device->d3d12_caps.max_shader_model == D3D_SHADER_MODEL_6_7)
```

Comment:
Seems weird to not just fuse the previous commit with this one.

Reply:
done